lib/cryptoexchange/helpers/hash_helper.rb

```diff
@@ -0,0 +1,20 @@
+module HashHelper
+  class << self
+    # "Polyfill" of the Hash#dig method from Ruby 2.3
+    #
+    #   Examples
+    #
+    #     > dig({a: {b: 1}}, :a, :b)
+    #     => 2
+    #
+    #     > dig({a: {b: 1}}, :a, :c)
+    #     => nil
+    def dig(obj, *args)
```

Comment:
@tmlee actually you're correct, my blurry memory told me it was `2.4` but it is actually `2.3`. Going to merge this one